src/v2/Apps/Artwork/Components/PricingContextModal.tsx

```diff
@@ -59,19 +59,19 @@ export class PricingContextModal extends React.Component<State> {
           }
         >
           <Spacer mt={2} />
-          <Serif size="3" color="black80">
+          <Text variant="text" color="black80">
```

Comment:
**suggestion, (non-blocking)**: I believe `black80` is [deprecated](https://www.notion.so/artsy/Color-a0c24896daf8433d9409aee2146ac267#c19e89075a1e45c2b8640f6d22ff48ad)? Nbd but could change here and throughout this file.